### PR TITLE
Fix NWB write functions

### PR DESCRIPTION
**Pull request by alejoe91**

### CODE REVIEW

**Comment by alejoe91** [comment]
Should we add nwbfile_kwargs is None also to write_recordings?

**Comment by colehurwitz** [comment]
Potentially. Let’s wait on the NWB people responses

File: spikeextractors/extractors/nwbextractors/nwbextractors.py [hunk 1]
```diff
@@ -74,15 +75,31 @@ def __init__(self, file_path, acquisition_name=None):
...
 
-        nwbfile = NWBFile(
-            session_description='',
-            identifier='',
-            session_start_time=datetime.now(),
-        )
+        if os.path.exists(save_path):
+            os.remove(save_path)
+
+        if 'session_description' not in nwbfile_kwargs.keys():
```
Comment (bendichter) [inline spikeextractors/extractors/nwbextractors/nwbextractors.py]:
```suggestion
        if 'session_description' not in nwbfile_kwargs:
```

**Comment by bendichter** [comment]
This looks good to me. The original idea behind having nwbfile_kwargs be input an whole dict was to allow for kwargs for different neurodata_types, similar to the way seaborn passes args [here](http://seaborn.pydata.org/generated/seaborn.jointplot.html#seaborn.jointplot). I think this is approach is fine though

**Comment by alejoe91** [comment]
> This looks good to me. The original idea behind having nwbfile_kwargs be input an whole dict was to allow for kwargs for different neurodata_types, similar to the way seaborn passes args [here](http://seaborn.pydata.org/generated/seaborn.jointplot.html#seaborn.jointplot). I think this is approach is fine though

Also in seaborn the input is **kwargs so it can be a dictionary or any set of extra keywords. I found a bug in setting the sampling frequency by adding tests (it was multiplying by 1000, when both are in Hz).

Also the SortingExtractor is not implemented. Can you help us including it? See https://github.com/SpikeInterface/spikeextractors/issues/284

File: spikeextractors/extractors/nwbextractors/nwbextractors.py (hunk 1 above)
Comment (alejoe91) [reply spikeextractors/extractors/nwbextractors/nwbextractors.py]: :+1:

**Pull request merged**